api/src/main/java/edu/cornell/mannlib/vitro/webapp/visualization/personpubcount/CumulativeCountRequestHandler.java

```diff
@@ -121,7 +122,14 @@ public Object generateAjaxVisualization(VitroRequest vitroRequest, Log log, Data
         csv.append("Year,Previous,Other");
         // Add the configured groups to CSV headings
         for (String groupName : pubsGroups) {
-            csv.append(",").append(groupName);
+            // Retrieve a label from the i18n file
+            String label = I18n.text(vitroRequest, "histogram_label_for_" + groupName);
+            if (!StringUtils.isEmpty(label)) {
```

Comment:
Simple fix for the above: https://github.com/vivo-project/VIVO/commit/98cdea6f68a7fbc143f4d36b9b5509353861f2ee

Reply:
That's not quite true - the test is for an empty string, which could be what is present in the text bundle.

I'm inclined not to change this - the obvious way to fix a text bundle error being displayed is to add the key to a text bundle. The text being returned when the key isn't present isn't (afaik) meant to be a contract, so testing it may be fragile if the text retrieval method is changed.

Reply:
Ah right, you could have an empty string in the i18n file. So in its current state, the code will only default to the group name in runtime.properties if the key is added to an i18n file, but without a label. That seems unlikely to happen but I suppose the argument is the proper use will be documented and returning the error mirrors the behavior for any other i18n string.